source/grammar/qasm3.g4

```diff
@@ -470,7 +503,7 @@ pragma
 
 timingType
     : 'length'
-    | 'stretch' Integer?
+    | StretchN
```

Comment:
Please note the other PR's this is pulling in; will make it easier to review. Let's also focus on getting those merged in first

Reply:
This is in #158. It can be merged soon, I believe.